source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml

```diff
@@ -116,24 +116,34 @@
         </td>
         <!-- /ko -->
         <td>
-          <div data-bind="visible: hasComments " class="comment">
+          <div class="comment">
             <label data-bind="css: { 'checked': showComments()}">
               <input class="cmn-toggle-round" type="checkbox" data-bind="checked: showComments" />
             </label>
           </div>
         </td>
       </tr>
       <tr class="comments-section" data-bind="visible: showComments">
-        <td colspan="4">
+        <td data-bind="attr:{'colspan': $root.columnsAmount}">
           <!-- ko foreach: comments -->
           <div class="comments-section-container" data-bind="visible: HasComment">
-            <div>
-              <div class="comment-author"><span data-bind="text: evaluatorEmployee"></span></div>
-              <div class="comment-author"><p>:</p></div>
+            <span class="comment-author" data-bind="text: evaluatorEmployee"></span><sapn class="comment-author">:</sapn>
+            <span class="comment-text" data-bind="text: value, visible: !isSelfComment || !IsEditingComment()"></span>
+            <div class="comment-text" data-bind="visible: IsEditingComment" style="float:left; width:100%">
+              <textarea data-bind="value: value" rows="5" style="width: 98%; resize: vertical" placeholder="Agrega un comentario"></textarea>
```

Comment:
_Percentage widths_ are problematic, did you tried other alternatives like margin?  

Also, did you consider to move styles to a CSS file?